src/nodemanagerbase.cpp

```diff
@@ -27,6 +27,8 @@
 #include <stdexcept>
 #include <uadatavariablecache.h>
 
+static UaLocalizedText emptyDescription( "en_US", "" );
+
 NodeManagerBase::NodeManagerBase( const char* uri, bool sth, int hashtablesize ):
     m_server(0),
     m_nameSpaceUri(uri)
```

Comment:
Actually refers to line 61 (I don't know how to add comments to lines that the diff doesn't show).

The refactor could also ditch BOOST_FOREACH too I guess, now we're C++11ish.

Reply:
or maybe std::find_if